aten/src/ATen/FunctionalTensorWrapper.cpp

```diff
@@ -322,6 +322,57 @@ void sync(const c10::List<c10::optional<Tensor>> t_list) {
   }
 }
 
+bool isFunctionalTensor(const at::Tensor& tensor) {
```

Comment:
TORCH_API missing here I think as it is in the header

Reply:
meta question: Does TORCH_API need to be on the function definitions? I thought it was only necessary for function declarations

Reply:
(yep I think you're right and I added it unnecessarily in a few places - you should need it in the declarations in the header files, but not in the actual cpp files)